lms/djangoapps/courseware/features/video.feature

```diff
@@ -1,32 +0,0 @@
-@shard_2 @requires_stub_youtube
-Feature: LMS.Video component
-  As a student, I want to view course videos in LMS
-
-  # 1
-  Scenario: Verify that each video in sub-section includes a transcript for Youtube and non-Youtube countries
-    Given youtube server is up and response time is 2 seconds
-    And I am registered for the course "test_course"
-    And I have a "subs_3_yD_cEKoCk.srt.sjson" transcript file in assets
-    And I have a "subs_b7xgknqkQk8.srt.sjson" transcript file in assets
-    And I have a "chinese_transcripts.srt" transcript file in assets
-    And it has videos "A, B" in "Youtube_HTML5" mode in position "1" of sequential:
```

Comment:
Display of a video when youtube response is 2 seconds (forcing it to default to HTML) is covered by 
https://github.com/edx/edx-platform/blob/master/common/test/acceptance/tests/video/test_video_module.py#L372

Verification of youtube transcript display when youtube is disabled is covered by
https://github.com/edx/edx-platform/blob/master/common/test/acceptance/tests/video/test_video_module.py#L433